src/connection/eagerlyConnect.ts

```diff
@@ -20,8 +20,14 @@ async function connect(connector: Connector, type: ConnectionType) {
   }
 }
 
-connect(gnosisSafeConnection.connector, ConnectionType.GNOSIS_SAFE)
+// The Safe connector will only work from safe.global, which iframes;
```

Comment:
can you give me some more context on this?

Reply:
[safe app](https://app.safe.global/welcome) works by iframing dapps, and this is a special connector that proxies RPCs over the iframe messaging API (`sendMessage`). If we're not in an iframe, we're also not in the safe app, so this connector (and its baggage) are unnecessary.